src/store/pipeline-report-sink.ts

```diff
@@ -53,16 +53,19 @@ export class PipelineReportSink implements ReportSink {
         log.info(`Saving report using ${name}...`);
         const maybeReportInfo = await sink.saveReport(lastReportInfo);
 
-        // Pass returned report info to the next sink
-        if (maybeReportInfo !== undefined) {
-          lastReportInfo = maybeReportInfo;
-          log.info(`Report saved using ${name}`, {
-            ...lastReportInfo,
-            report: undefined,
-          });
+        if (!maybeReportInfo) {
+          return;
         }
+
+        // Pass returned report info to the next sink
+        lastReportInfo = maybeReportInfo;
+        log.info(`Report saved using ${name}`, {
+          ...lastReportInfo,
+          report: undefined,
```

Comment:
> IF we want to log report as undefined because it's too big, why not just omit report and log only ReportInfo?

That's essentially what we're doing. `ReportInfo` contains `report`. `report: undefined` is removing it.

Reply:
I was just initially confused why were explicitly setting it when I looked back at this, so I figured a comment was warranted. :slightly_smiling_face:

Reply:
I mean, me as a user, I rather have no `report` in the log as having as `undefined`. I would read that as an issue.

Reply:
The `undefined` doesn't get logged.